app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,99 @@
+class FriendshipsController < ApplicationController
+  include LolConnections
+  before_action :search_summoner_for_request_action, only: [:edit]
+  before_action :require_user
+  before_action :require_same_user, only: [:edit, :decide]
+  
+ 
+	def new
+    @user = current_user
```

Comment:
Yes, in practice we should actually be storing that user variable completely somewhere else since it is used by multiple controllers.  I think this is fine for now.  It is generally alright to use instance variables for reasons other than passing info to views as long as it is a good reason.
+1